contrib/fwupd.spec.in

```diff
@@ -1,6 +1,6 @@
 %global glib2_version 2.45.8
 %global libxmlb_version 0.1.3
-%global libgusb_version 0.3.3
+%global libgusb_version 0.3.5
```

Comment:
Although this is correct I think you'll have to hold off doing this unless 0.3.5 is really landing in fedora soon.

Reply:
It's already there, no?

![Screenshot_2020-08-26 Fedora Updates System](https://user-images.githubusercontent.com/151380/91277299-8b75c900-e77a-11ea-8db8-facc161ef6e7.png)

Reply:
Oh I see the problem - that's for fedora 32.  We run fedora 31 in CI.

Reply:
Want me to bump now as a separate PR?

Reply:
Yeah I think that makes sense if you can.  Then this can just rebase on master.

Reply:
https://github.com/fwupd/fwupd/pull/2337

Reply:
Thanks a bunch, guys.